### PR TITLE
add links to next step info to learn more page to become a counselor

### PR DESCRIPTION
**Pull request by saragibby**
Made some changes to make this page a bit more visual and personalized to the state of the user. Items changed:
- added bear
- button links based on state rather than deferring to session/create to redirect
- adds FAQ link
- adds next step text personalized to current state of user (not signed in, no profile, signed in w/ profile)

### When not signed in
image

### Signed in but no profile
image

### Signed in, has profile
image

### CODE REVIEW

**Comment by csell5** [comment]
#### 📝Changed routes:


| [**/wi/counselor-start**](https://that-website-git-sg-add-session-faq.that-conference.now.sh/wi/counselor-start) |
|:-:|
|<a href="https://that-website-git-sg-add-session-faq.that-conference.now.sh/wi/counselor-start">Screenshot of /wi/counselor-start</a><br /><sup><a href="https://images.weserv.nl?url=https://microlink.nyc3.digitaloceanspaces.com/IHXT5YyKPy6N1DoRezmGYdzzqWU_">(view full size)</a> |







Commit f69cd296611ef326cac7139d9bbad6872c1ccb5f (https://that-website-git-sg-add-session-faq.that-conference.now.sh).

File: markdown/counselor-submission-faq.md [hunk 1]
```diff
@@ -3,24 +3,26 @@ title: Session Submission FAQs
...
 
 After THAT Conference 2019, we deleted our old site and created a new one from the ground up. Everything went with it, including user accounts. You have to create a new account on the site.
 
 ### What happened to my old sessions?
 
-After THAT Conference 2019, we deleted our old site and created a new one from the ground up. Everything went with it, including sessions. We may decide to migrate the old data over to the new site and connect it with your profile. Things are still under development so we're not 100% sure what will happen.
+After THAT Conference 2019, we deleted our old site and created a new one from the ground up. Everything went with it, including sessions. We are currently working to build out the rest of the site functionality, including migrating over past sessions and speakers. [Sign up for our newsletter](#newsletter), [join THAT Slack](https://thatslack.thatconference.com/) and/or watch our social media as we announce new site functionality as we release it.
+
+We may decide to migrate the old data over to the new site and connect it with your profile. Things are still under development so we're not 100% sure what will happen.
```
Comment (csell5) [inline markdown/counselor-submission-faq.md]: How about "Things are still under development so we're not 100% sure the exact timelines or details on the migration."

Comment (csell5) [inline markdown/counselor-submission-faq.md]: How about. "Things are still under development so we're not 100% sure what will happen."

Comment (saragibby) [reply markdown/counselor-submission-faq.md]: @csell5 wait, my bad, I meant to remove that line. You rather keep + reword it, or remove it?

Comment (csell5) [reply markdown/counselor-submission-faq.md]: ah yea.. just remove it. less is more.

**Pull request merged**